ssn/index.html

```diff
@@ -7604,6 +7604,20 @@ <h3>Changes since W3C Candidate Recommendation 11 July 2017 <a href="https://www
           <li>Removed Features at Risk section</li>
           <li>Removed Candidate Recommendation Exit Criteria section</li>
         </ol>
+        <h3>Changes implementing <a href="https://www.w3.org/2017/10/vocab-ssn-errata.html">Errata #6</a> since W3C Recommendation 19 October 2017 <a href="https://www.w3.org/TR/2017/REC-vocab-ssn-20171019/">
```

Comment:
It's a good idea to maintain the "changes" section. Instead of adding an `<h3>` entry for all changes we will be making, I would suggest to have a "Changes since W3C Recommendation 19 October 2017" and list changes that have been made to the document on top of relaxing axioms too. Typically, I propose:

```html
<h3>Changes since W3C Recommendation 19 October 2017 <a href="https://www.w3.org/TR/2017/REC-vocab-ssn-20171019/">(https://www.w3.org/TR/2017/REC-vocab-ssn-20171019/)</a></h3>
<ol>
  <li>Fixed remaining broken links to ontology graph and namespaces and "specifically" typo</li>
  <li>Fixed Seismographs example, replacing the final <code>;</code> with a <code>.</code></li>
  <li>Fixed targets of <code>sosa:hosts</code> and <code>ssn:hasSubSystem</code> in DHT Deployment example</li>
  <li>Fixed domain/range for <code>actsOnProperty</code> and <code>isActedOnBy</code> in actuation diagrams</li>
  <li>Replaced inexistant <code>sosa:actuationMadeBy</code> by <code>sosa:madeByActuator</code> in apartment 134 example</li>
  <li>Relaxed axioms in 6.2 SSNX Alignment Module:
    <ul>
      <li>oldssn:FeatureOfInterest is a sub-class of sosa:FeatureOfInterest (instead of equivalent)</li>
      <li>oldssn:hasValue is a sub-property of sosa:hasResult (instead of equivalent)</li>
      <li>oldssn:observationResult is a sub-property of sosa:hasResult (instead of equivalent)</li>
    </ul>
  </li>
  <li>Relaxed axioms in 6.1 Dolce-Ultralite Alignment Module:
    <ul>
      <li>sosa:FeatureOfInterest is a sub-class of dul:Entity</li>
    </ul>
  </li>
</ol>
```